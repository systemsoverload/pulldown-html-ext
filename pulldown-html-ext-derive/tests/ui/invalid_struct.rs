use pulldown_html_ext_derive::html_writer;

struct HtmlWriterBase<W>(W);

#[html_writer]
enum InvalidEnum {  // This should fail
    Variant(HtmlWriterBase<String>),
}

fn main() {}
