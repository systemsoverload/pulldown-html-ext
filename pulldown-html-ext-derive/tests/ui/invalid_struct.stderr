error: html_writer can only be applied to structs, not enums
 --> tests/ui/invalid_struct.rs:6:6
  |
6 | enum InvalidEnum {  // This should fail
  |      ^^^^^^^^^^^
